.bazelrc

```diff
@@ -75,10 +75,10 @@ build:java19 --tool_java_language_version=19
 build:java19 --java_runtime_version=remotejdk_19
 build:java19 --tool_java_runtime_version=remotejdk_19
 
-build:java20 --java_language_version=20
-build:java20 --tool_java_language_version=20
-build:java20 --java_runtime_version=remotejdk_20
-build:java20 --tool_java_runtime_version=remotejdk_20
+#build:java20 --java_language_version=20
+#build:java20 --tool_java_language_version=20
+#build:java20 --java_runtime_version=remotejdk_20
+#build:java20 --tool_java_runtime_version=remotejdk_20
```

Comment:
You're going to need to retain Java 20 support.

Reply:
Indeed, except see above. I don't intend to merge this though.

Actually my high-level question here is whether y'all would care about an approach such as this.  Such high level feedback would be most welcome.

I worked out the "her" builds with bazel which enable doing some really cool stuff with bazel, since one longer needs to bazelize the world, and one is no longer required to start praying to the Nix gods.

Applying it to `kythe` hopefully shows that her approach has merit.

But if y'all don't think this matters, I'm happy to retreat to my cave and perhaps use the approach for other stuff.

Reply:
As I'm not at Google anymore, I'm not in a position to say, but it seems like a beneficial change.

Hopefully someone on the current team will be able to engage with you on the change, though.

Reply:
Reverted.